### ISSUE TITLE
Afer GemTalk/Rowan#428 fixed, Service tests for `ProjectNotOnDisk` failing ...

### ISSUE THREAD

**Issue by dalehenrich** [ericwinger/Jade#333]
Not surprisingly, the two`ProjectNotOnDisk` tests are failing, now that semantics for #existsOnDisk has changed ... since there appears to be test coverage over and above simpe disk presence, I'll let @ericwinger decide how he wants to refactor the tests:
  ```
    Rowan STON Cypress Tonel tests for GemStone '3.2.15'
    584 run, 575 passed, 4 failed, 5 errors
      errors
        RwBrowserToolApiTest>>#testMoveGlobalExtensionSessionMethods
        RwBrowserToolApiTest>>#testNewClassVersion_session_method_unchanged_extension_method_protocol
        RwHybridBrowserToolTest>>#testHybridMoveMethodFromSessionMethodsIntoSessionMethods
        RwHybridBrowserToolTest>>#testHybridMoveMethodIntoSessionMethods
        RwRowanSample2Test>>#testNoMigration_gitolite
      failures
-->        RowanClassServiceTest>>#test_addedProjectNotOnDisk
-->        RowanProjectServiceTest>>#test_addedProjectNotOnDisk
        RwLoadingTest>>#testPoolDictionaryChanges
        RwSymbolDictionaryTest>>#testClassVersioningPatch
```
[This commit](https://github.com/GemTalk/Rowan/commit/c8df6fe9227f7f6fcffdf8e8fd3541265bdda691) (Rowan tests for GemTalk/Rowan#428) has sample code that creates a project that is not written to disk and one that is written to disk ...

**Comment by dalehenrich** [comment]
Added `P1-critical` label, since these failing tests should be passing before next v1.2.x release ... @ericwinger, or @LisaAlmarode, ff there's a better way to mark this issue, let me know.

**Comment by ericwinger** [comment]
Fixed in Oscar 3.0.56 on the server. https://github.com/GemTalk/Rowan/commit/e22468cbc873a87f72f90ab459bb13500774c556
